--- icu/source/common/unicode/rbbi.h.setBreakType	2012-04-05 22:46:08.000000000 +0200
+++ icu/source/common/unicode/rbbi.h	2012-04-07 19:18:41.629484616 +0200
@@ -658,6 +658,11 @@ public:
      */
     virtual RuleBasedBreakIterator &refreshInputText(UText *input, UErrorCode &status);
 
+    /**
+      * Set the type of the break iterator.
+      * @internal
+      */
+    virtual void setBreakType(int32_t type);
 
 protected:
     //=======================================================================
@@ -688,12 +693,6 @@ protected:
     virtual int32_t getBreakType() const;
 #endif
 
-    /**
-      * Set the type of the break iterator.
-      * @internal
-      */
-    virtual void setBreakType(int32_t type);
-
 #ifndef U_HIDE_INTERNAL_API
     /**
       * Common initialization function, used by constructors and bufferClone.
