--- icu/source/common/unicode/rbbi.h.0000~	2016-11-12 02:44:37.804432639 +0100
+++ icu/source/common/unicode/rbbi.h	2016-11-12 02:46:16.356195580 +0100
@@ -638,6 +638,12 @@ public:
      */
     virtual RuleBasedBreakIterator &refreshInputText(UText *input, UErrorCode &status);
 
+    /**
+      * Set the type of the break iterator.
+      * @internal
+      */
+    void setBreakType(int32_t type);
+
 
 private:
     //=======================================================================
@@ -651,12 +657,6 @@ private:
     void reset(void);
 
     /**
-      * Set the type of the break iterator.
-      * @internal
-      */
-    void setBreakType(int32_t type);
-
-    /**
       * Common initialization function, used by constructors and bufferClone.
       * @internal
       */
